src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
+@import './utils/variables';
+@import './blocks/stars';
+@import './blocks/card';
+
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  line-height: 18px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), font styles should be in the `html` selector, not `body`. Move `font-family`, `font-weight`, and `line-height` properties to the `html` selector.